storage/src/vespa/storage/common/storagelink.cpp

```diff
@@ -165,9 +165,9 @@ void StorageLink::sendUp(const shared_ptr<StorageMessage> & msg)
         default:
             LOG(error, "Link %s trying to send %s up while in state %s",
                 toString().c_str(), msg->toString(true).c_str(), stateToString(getState()));
-            assert(false);
+            return;
```

Comment:
Is it safe to remove these abort-branches? If the link is not in a state where it's safe to process certain types of messages, the behavior is likely undefined

Reply:
It was initially removed, as there was a test failing in communication manager. However there is no longer need for this as the test itself was fixed.